packages/modules/viewer-react/src/tests/components/iModel/IModelViewer.test.tsx

```diff
@@ -10,6 +10,7 @@ import { FrontstageProvider } from "@itwin/appui-react";
 import { UiFramework } from "@itwin/appui-react";
 import { render } from "@testing-library/react";
 import React from "react";
+import { createRoot } from "react-dom/client";
```

Comment:
where are you using this?

add this in the web-test app, comment out the ui providers for the time being, make sure strict mode is enabled and try to go through all the issues with the hooks

Reply:
removed, no longer using